### PR TITLE
docs(removing presets): removing presets chapter has been added

### PR DESCRIPTION
**Pull request by dbarczewski**

### CODE REVIEW

File: docs/RemovingPresets.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+1. [Introduction](#introduction)
+2. [FSXA-UI](#fsxa-ui)
+3. [TailwindCSS](#tailwindcss)
+4. [Vue-TSX-Support](#vue-tsx-support)
+5. [Semantic-Release](#semantic-release)
+
+### Introduction
+
+This chapter introduces optional dependencies. Why we have them in our project and how you can easily remove them.
+### FSXA-UI
```
Comment (vlapl) [inline docs/RemovingPresets.md:14]: The introduction currently grammatically really only is one sentence (either change '.' to ',', followed by 'why', or ajust the second sentence.

File: docs/RemovingPresets.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+The [FSXA-UI](https://github.com/e-Spirit/fsxa-ui) is a component library containing all sections that have been implemented in our project 'Smart Living'.
+
+#### Why do we use this?
+
+We use this component library to display the individual components in the PWA.
+We do not have to implement them here, but provide them with a detailed documentation.
+
```
Comment (vlapl) [inline docs/RemovingPresets.md:22]: The second sentence is not clear to me. Does it mean we can implement components elsewhere (like in the pwa project directly), but still can/ 'have to' document them in this project? That seems strange to me.

File: docs/RemovingPresets.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+```shell
+npm uninstall fsxa-ui
+```
+
+The FSXA UI must be removed from all components in the `components/fsxa` folder.
+
+The best way to see if a component is used is to look at the imports at the beginning of a file.
+If the FSXA-UI is imported there, this import and from the `render` function the component must be removed.
+
```
Comment (vlapl) [inline docs/RemovingPresets.md:35]: Better: "If the FSXA-UI is imported there, remove the import. Also remove any elements from FSXA-UI used in the `render` function."

File: docs/RemovingPresets.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+Also, the CSS file `'fsxa-ui/dist/fsxa-ui.css'` must be removed from the `css` attribute from the `nuxt.config.ts` file.
+
+### TailwindCSS
+
+[TailwindCSS](https://tailwindcss.com/) is a utility-first CSS framework. With which the writing of own CSS classes is no longer necessary.
+#### Why do we use this?
```
Comment (vlapl) [inline docs/RemovingPresets.md:41]: Like with the fsxa-ui introduction: "With which" does not (gramatically) start a new sentence - join or reword.

File: docs/RemovingPresets.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+We use TailwindCSS to create responsive designs without having to create additional CSS files.
+
+#### How to remove
+
+First, TailwindCSS and the required dependencies must be removed.
```
Comment (vlapl) [inline docs/RemovingPresets.md:47]: In the ticket description Tobi wrote "tailwindcss (just link to the remove tailwind css page)". That would be [RemovingTailwind.md](https://github.com/e-Spirit/fsxa-pwa/blob/documentation/docs/css/RemovingTailwind.md) I guess.

File: docs/RemovingPresets.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+```shell
+npm uninstall @nuxtjs/tailwindcss tailwindcss @tailwindcss/postcss7-compat postcss autoprefixer
+```
+
+Also, in `nuxt.config.ts` for the `buildModules` attribute, the `'@nuxtjs/tailwindcss'` entry must be removed.
+
+Attention: By removing TailwindCSS, the classes used in the components are of course no longer present, so the styling will now look different.
+
```
Comment (vlapl) [inline docs/RemovingPresets.md:55]: Maybe add quotation marks to 'different', to indicate some irony? ;-) Without tailwind the layout will hardly be recognizable anymore, I suppose...

File: docs/RemovingPresets.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+### Vue-TSX-Support
+
+[Vue-Tsx-Support](https://github.com/wonderful-panda/vue-tsx-support) is a library that allows to use the TSX syntax with Vue.js.
+
+#### Why do we use this?
+
+We use this library to get better TypeScript support.
+
```
Comment (vlapl) [inline docs/RemovingPresets.md:63]: Why 'better'? Wouldn't TypeScript support be gone completely, without this?

File: docs/RemovingPresets.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+#### How to remove
+
+First, all the required dependencies must be removed.
+```shell
+npm uninstall cz-conventional-changelog commitizen @semantic-release/changelog @semantic-release/commit-analyzer @semantic-release/git @semantic-release/github @semantic-release/release-notes-generator @commitlint/cli @commitlint/config-conventional
+```
+
+Also, the scripts need to be removed.
+<br />
+To do this, the entries `commit`, `commitmsg` and `semantic-release` must be removed in `package.json` under the attribute `scripts`.
```
Comment (vlapl) [inline docs/RemovingPresets.md:91]: Again, missing comma before 'and' in the list. The English rule for this differs from the German grammar.

File: docs/RemovingPresets.md (hunk 2 above)
Comment (dbarczewski) [reply docs/RemovingPresets.md:22]: I must agree with you, but does not have an idea how to write this better. Let's talk about this.

File: docs/RemovingPresets.md (hunk 7 above)
Comment (dbarczewski) [reply docs/RemovingPresets.md:63]: We would not have TypeSupport when we write our HTML templates. We would still be able to use TS-files in our project.

**Pull request merged**